example_multitenant_update_theme.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+CONFIGURATION_PLAYBOOKS_DIR='/encrypted/ansible/edx/playbooks'
+THEMEX_SCRIPT_DIR='/encrypted/ansible/edx/playbooks/themex_script'
```

Comment:
It's not always /encrypted. It can be variable and use default value if not set.